packages/core/admin/admin/src/pages/SettingsPage/components/SettingsNav/index.js

```diff
@@ -35,14 +37,26 @@ const SettingsNav = ({ menu }) => {
     defaultMessage: 'Settings',
   });
 
+  const handleClick = (link) => {
+    if (link.to.startsWith('/settings/audit-logs')) {
+      trackUsage('didGoToAuditLogs');
+    }
+  };
+
```

Comment:
Could you circle this back and check if we can use 

```
trackUsage('willNavigate', { from: pathname, to: destination });
```

here too? This was introduced in https://github.com/strapi/strapi/pull/14875/files and at the time we decided we don't want individual events for every item, but one event instead that gives us deep insight into navigations.

Reply:
Approved by Yannis 👍